framework/doc/content/source/materials/CoupledGradientMaterial.md

```diff
@@ -0,0 +1,17 @@
+# CoupledValueFunctionMaterial / ADCoupledGradientMaterial
+
+!syntax description /Materials/CoupledGradientMaterial
+
+This material provides the ability to create a material property from the gradient of a variable. In addition, the gradient can be scaled with a scalar material property allows for additional flexability. This gradient material property can then be used in kernels such as [ADConservativeAdvection](/ADConservativeAdvection.md).
```

Comment:
```suggestion
This material provides the ability to create a material property from the gradient of a variable. In addition, the gradient can be scaled with a scalar material property allows for additional flexibility. This gradient material property can then be used in kernels such as [ADConservativeAdvection](/ADConservativeAdvection.md).
```

Reply:
why conservative advection? It expects a velocity vector material property but that's not the gradient of anything? 
Or are you thinking about about Darcy law where v = D grad P?